apache-rat-core/src/main/java/org/apache/rat/license/LicenseSetFactory.java

```diff
@@ -338,7 +344,7 @@ public SortedSet<String> getLicenseCategories(final LicenseFilter filter) {
                 return result;
             case APPROVED:
                 approvedLicenseCategories.stream().filter(s -> !removedLicenseCategories.contains(s)).forEach(result::add);
-                licenses.stream().filter(approved).forEach(l -> result.add(l.getLicenseFamily().getFamilyCategory()));
+                //licenses.stream().filter(approved).forEach(l -> result.add(l.getLicenseFamily().getFamilyCategory()));
```

Comment:
@Claudenw is this a leftover or a TODO ?

Reply:
I removed the piece of code via [16a2319](https://github.com/apache/creadur-rat/pull/413/commits/16a2319181f4a60f414f04e42ab1932ec94207f6)